feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/data/thresold/gov1/Gov1DelayedThreshodPassing.kt

```diff
@@ -0,0 +1,25 @@
+package io.novafoundation.nova.feature_governance_api.data.thresold.gov1
+
+import io.novafoundation.nova.common.data.network.runtime.binding.Perbill
+import io.novafoundation.nova.feature_governance_api.data.network.blockhain.model.DelayedThresholdPassing
+import io.novafoundation.nova.feature_governance_api.data.network.blockhain.model.Tally
+import io.novafoundation.nova.feature_governance_api.data.network.blockhain.model.VotingThreshold
+import io.novafoundation.nova.feature_wallet_api.data.network.blockhain.types.Balance
+import java.math.BigDecimal
+
+class Gov1DelayedThresholdPassing(
+    val threshold: VotingThreshold
+) : DelayedThresholdPassing {
+
+    override fun supportPassingInFuture(tally: Tally, totalIssuance: Balance, passedSinceDecidingFraction: Perbill): DelayedThresholdPassing.DelayedPassing {
+        val supportThreshold = threshold.supportThreshold(tally, totalIssuance, passedSinceDecidingFraction)
+
+        return DelayedThresholdPassing.DelayedPassing(BigDecimal.ONE, supportThreshold.currentlyPassing)
+    }
+
+    override fun ayePassingInFuture(tally: Tally, totalIssuance: Balance, passedSinceDecidingFraction: Perbill): DelayedThresholdPassing.DelayedPassing {
+        val approvalThreshold = threshold.ayesFractionThreshold(tally, totalIssuance, passedSinceDecidingFraction)
+
+        return DelayedThresholdPassing.DelayedPassing(BigDecimal.ONE, approvalThreshold.currentlyPassing)
+    }
+}
```

Comment:
Gov1 uses this class more as a placeholder. Because this implementation is not used